docs/cpp/cmake-linux.md

```diff
@@ -0,0 +1,176 @@
+
+The VS Code CMake Tools extension does its work using CMake installed on your system. For best results, use CMake version 3.17 or greater.
+
+See if CMake is already installed on your system. Open a Terminal window and enter the following command:
+
+```bash
+cmake --version
+```
+
+To install, or get the latest version of CMake on your system, see the instructions for your platform on the [Kitware APT Repository](https://apt.kitware.com/). You want to install a version greater than 3.17.
+
+### Ensure that development tools are installed
+
+Although you'll use VS Code to edit your source code, you'll compile and debug the source code using the compiler, debugger, and build tools (such as make) installed on your system.
+
+For this tutorial on Ubuntu, we'll use the GCC compiler, GDB to debug, and make to genenerate native build files. These tools are not installed by default on Ubuntu, so you need to install them. Fortunately, that's easy.
+
+### Check if GCC is installed
+
+ To see if GCC is already installed on your system, open a Terminal window and enter the following command:
+
+```bash
+gcc -v
+```
+
+If GCC isn't installed, run the following command from the terminal window to update the Ubuntu package lists. An out-of-date Linux distribution can interfere with getting the latest packages.
+
+```bash
+sudo apt-get update
+```
+
+Next, install the GNU compiler, make, and the GDB debugger with this command:
+
+```bash
+sudo apt-get install build-essential gdb
+```
+
+## Create a CMake quickstart
+
+If you have an existing CMake project that already has a CMakeLists.txt file in the root directory, you can skip to [Select a kit](#select-a-kit) to configure your existing project.
+
+Otherwise, create a folder for a new project. From the terminal window, create an empty folder called `cmakeQuickStart`, navigate into it, and open VS Code in that folder by entering the following commands:
+
+```cmd
+mkdir cmakeQuickStart
+cd cmakeQuickStart
+code .
+```
+
+The `code .` command opens VS Code in the current working folder, which becomes your "workspace".
+
+### Create a CMake hello world project
+
+The CMake Tools extension can create the files for a basic CMake project for you. Open the Command Palette (`kb(workbench.action.showCommands)`) and run the **CMake: Quick Start** command:
+
+![Create CMake quickstart](images/cpp/cmake-quickstart-command-palette.png)
+
+Enter a project name. This will be written to `CMakeLists.txt` and a few initial source files.
+
+Next, select **Executable** as the project type to create a basic source file (`main.cpp`) that includes a basic `main()` function.
+
+![Choose project type](images/cpp/cmake-choose-type.png)
+
+**Note:** If you had wanted to create a basic source and header file, you would have selected **Library** instead. But for this tutorial, **Executable** will do. If you are prompted to configure IntelliSense for the folder, you can either ignore the prompt or select **Allow**.
```

Comment:
People should not ignore the prompt. They will only get the best IntelliSense experience if they click the "Allow" button.

Reply:
fixed